File: Python/Estrutura repetitiva/soma.py
# Variável
troca = 0
soma = 0

# Entrada de dados
num1 = int(input("Digite um número: "))
num2 = int(input("Digite outro número: "))

# Estrutura condicional
if (num1 > num2):
    troca = num1
    num1 = num2
    num2 = troca

# Estrutura repetitiva
for i in range(num1+1, num2): # Tirei as extremidades
    # Estrutura condicional
    if (i % 2 != 0):
        soma = soma + i

# Saída de dados
print(f"A soma dos ímpares de {num1} e {num2} é {soma}.")

File: Java/Estrutura repetitiva/soma.java
// Biblioteca
import java.util.Scanner;

public class soma {

	public static void main(String[] args) {
		// Variável
		int troca, soma, i, num1, num2;
		
		soma = 0;
		
		// Para facilitar a leitura
		Scanner sc = new Scanner(System.in);
		
		// Entrada de dados
		System.out.print("Digite um número: ");
		num1 = sc.nextInt();
		
		System.out.print("Digite outro número: ");
		num2 = sc.nextInt();
		
		// Estrutura condicional
		if (num1 > num2) {
			troca = num1;
			num1 = num2;
			num2 = troca;
		}
		
		// Estrutura repetitiva
		for (i = num1+1; i < num2; i++) {
			// Estrutura condicional
			if (i % 2 != 0) {
				soma += i;
			}
		}
		
		// Saída de dados
		System.out.print("A soma dos ímpares é " + soma);
		
		sc.close(); // Boa prática de programação
	}
}